.gitattributes

```diff
@@ -11,7 +11,7 @@
 ##   Handle line endings automatically for files detected as
 ##   text and leave all files detected as binary untouched.
 ##   This will handle all files NOT defined below.
-* text=auto
+* text=auto eol=lf
```

Comment:
Can you explain what this code does?

Reply:
This changes the line endings of all text files to `LF` (unix's default line endings). So, even if you're creating files on Windows or macOS, which uses `CRLF` & `CR` line endings respectively by default, this will change the line endigns to `LF` in the repo.
So, it doesn't matter what line endings you use in your dev env, it'll be consistent in the repo.